src/adapter/src/coord/sequencer.rs

```diff
@@ -3386,6 +3397,116 @@ impl<S: Append + 'static> Coordinator<S> {
         self.transient_id_counter += 1;
         Ok(GlobalId::Transient(id))
     }
+
+    /// TODO(jkosh44) This function will verify the privileges for the mz_introspection user.
+    ///  All of the privileges are hard coded into this function. In the future if we ever add
+    ///  a more robust privileges framework, then this function should be replaced with that
+    ///  framework.
+    fn mz_introspection_user_privilege_hack(
```

Comment:
Is it in bad taste to use the work "hack" in the method name? I wanted to convey to whoever was reading this that they can replace it with something better when that time comes.

Reply:
Not at all! This is great!